File: problems/b/bus/bus.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <bits/stdc++.h>

using namespace std;

int main() {
    int t, stops;

    cin >> t;
    cin.ignore();

    for(int i=0; i<t; i++) {
        cin >> stops;
        cin.ignore();
        int current_number_passengers = 0;
        for(int j=0; j<stops; j++) {
            current_number_passengers = (current_number_passengers + 0.5) * 2;
        }
        cout << current_number_passengers << endl;
    }
}

File: problems/b/bus/bus.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    for _ in range(int(input())):
        stops = int(input())
        current_number_passengers = 0
        for _ in range(stops):
            current_number_passengers = (current_number_passengers + 0.5) * 2

        print(int(current_number_passengers))


if __name__ == '__main__':
    main()